src/pages/Embeds/Plio.vue

```diff
@@ -1043,23 +1137,26 @@ export default {
       this.lastCheckTimestamp = timestamp;
       if (!this.isAspectRatioChecked) this.checkAndSetPlayerAspectRatio();
 
-      this.checkForItemPopup(timestamp);
+      const itemIndex = this.checkForItemPopup(timestamp);
+      if (itemIndex != null) {
+        this.currentItemIndex = itemIndex;
+        this.markItemSelected();
+        this.createEvent("item_opened", { itemIndex: this.currentItemIndex });
+      } else {
+        this.closeItemModal();
+      }
```

Comment:
doubt - why was this added?

Reply:
just for cleanup - I was earlier trying to use `checkForItemPopup` and it was doing more things than just checking for item popup - so, was hard to reuse - abhi although I didn't end up using `checkForItemPopup` as I was originally trying to, this cleanup still made sense, no?